magento/devdocs#: Layout instructions. Adding the example of disabling plugin. (#6856)
magento/devdocs: src/guides/v2.3/extension-dev-guide/plugins.md

For example, the developer can disable a global plugin in the [backend](https://glossary.magento.com/backend) area by disabling it in the specific `di.xml` file for the backend area.

### Disabling plugin.

> Each plugin can be disabling in a ``di.xml`` file. For it the ``disabled`` element should be set to ``true``

Reviewer: ```suggestion
Each plugin can be disabled in a ``di.xml`` file. To disable a plugin the ``disabled`` parameter of the plugin declaration should be set to ``true``
```

For example:

```xml
<type name="Magento\Checkout\Block\Checkout\LayoutProcessor">
    <plugin name="ProcessPaymentConfiguration" disabled="true"/>
</type>
```

> Where ``ProcessPaymentConfiguration`` is the name plugin declared in the `vendor/magento/module-payment/etc/frontend/di.xml` and is require parameter.

Reviewer: ```suggestion
Where ``ProcessPaymentConfiguration`` is the name of the plugin declared in the `vendor/magento/module-payment/etc/frontend/di.xml`.
```
